packages/core/admin/admin/src/content-manager/components/BlocksEditor/index.js

```diff
@@ -32,66 +31,80 @@ const Wrapper = styled(Box)`
   border-radius: ${({ theme }) => theme.borderRadius};
 `;
 
-const BlocksEditor = React.forwardRef(({ intlLabel, name, readOnly, required, error }, ref) => {
-  const { formatMessage } = useIntl();
-  const [editor] = React.useState(() => withReact(createEditor()));
+const BlocksEditor = React.forwardRef(
+  ({ intlLabel, name, readOnly, required, error, value, onChange }, ref) => {
```

Comment:
@remidej this is what caused the formatting diff. I added two new props which pushed this onto a new line and pushed everything nested over as well.